lib/pages/welcome_screen/welcome_screen.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_movie_clean/gen/assets.gen.dart';
+import 'package:flutter_movie_clean/gen/colors.gen.dart';
+import 'package:flutter_movie_clean/pages/welcome_screen/components/welcome_foreground.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class WelcomeScreen extends StatefulWidget {
+  const WelcomeScreen({super.key});
+
+  static const String routeLocation = "/welcome";
+  static const String routeName = "welcome";
+
+  @override
+  State<WelcomeScreen> createState() => _WelcomeScreenState();
+}
+
+class _WelcomeScreenState extends State<WelcomeScreen> {
+  late final PageController _pageController;
+  late final List<AssetGenImage> imageWelcomes;
+  int _currentIndex = 0;
+
+  @override
+  void initState() {
+    _pageController = PageController();
+    imageWelcomes = Assets.images.welcomes.values;
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    return Scaffold(
+      backgroundColor: AppColors.black,
+      body: SingleChildScrollView(
+        child: ConstrainedBox(
+          constraints: BoxConstraints(
+              minHeight: size.height,
+              maxHeight: size.height <=
+                      (num.parse((size.width / (360 / 720)).toStringAsFixed(1)))
+                  ? (size.width / (360 / 720))
+                  : size.height),
+          child: Stack(
+            children: [
+              Column(
+                children: [
+                  SizedBox(
+                    width: 360.w,
+                    height: 1.sh,
+                    child: PageView(
+                        onPageChanged: (index) => setCurrentIndex(index),
+                        controller: _pageController,
+                        children: imageWelcomes
+                            .map((e) => Column(
```

Comment:
Do we need to use ```Column``` here?
```suggestion
                            .map((e) => Column(
```